apps/site/src/lib/components/ValidatingForm/ValidatingForm.tsx

```diff
@@ -17,8 +17,8 @@ function ValidatingForm(props: PropsWithChildren<FormProps>) {
 		if (!form.checkValidity()) {
 			// prevent submission to display validation feedback
 			event.preventDefault();
+			setValidated(true);
 		}
-		setValidated(true);
```

Comment:
While this does technically work for our current use cases, it is not entirely flexible. Hypothetically, the form might want to show positive feedback for a valid field. I think the real problem is in the CSS selectors for the equivalent of `was-validated` and `invalid-feedback`.